client/wfprev-war/src/main/angular/src/app/components/list-panel/projects-list/projects-list.component.html

```diff
@@ -53,7 +53,7 @@
                 <img alt="detail-icon" class="icon" src="/assets/detail-icon.svg" />
                 <span>Details</span>
               </div>
-              <div class="action-item">
+              <div class="action-item" (click)="editProject(project)" #stopPropagation>
```

Comment:
What is #stopPropagation? I don't see it referenced elsewhere

Reply:
I use it to stop the click effect on expanding the panel. But I just moved this to typescript function "event.stopPropagation();" to handle it